internal/pkg/manifest/env.go

```diff
@@ -434,6 +434,7 @@ type PublicHTTPConfig struct {
 	SecurityGroupConfig ALBSecurityGroupsConfig `yaml:"security_groups,omitempty"`
 	Certificates        []string                `yaml:"certificates,omitempty"`
 	ELBAccessLogs       ELBAccessLogsArgsOrBool `yaml:"access_logs,omitempty"`
+	SecurityPolicy      *string                 `yaml:"security_policy,omitempty"`
```

Comment:
Can we make this consistent with https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/manifest/lb_web_svc.go#L231 so that both manifests have the field named [`ssl_policy`](https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#nlb-ssl-policy)`?

Reply:
yes, I can do that